pkgs/by-name/fr/freeshow/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  pkgs,
+  appimageTools,
+}:
+
+let
+  pname = "freeshow";
+  version = "1.2.1";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.dmg";
+          hash = "sha256-OO0uQ6oS1GKiBGz3Qt9jDCY+qdWlgTOY+SiLJB5xQ3c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.AppImage";
+          hash = "sha256-RBI8IkxY6Xd36vCVDJy9sqpEisB/48hwQo+mg5XTCOs=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
```

Comment:
Are we sure we can't build FreeShow from source?

Reply:
We probably could. but I don't see a remarkable benefit since upstream provided an appImage. It makes sure all the dependencies and libraries are hard coded into the package.Maybe the only advantage is that it would extend support to reach windows platforms.

Reply:
I'd rather reduce the amount of new pre-compiled packages on Nixpkgs, but this is a personal preference only, and the diff looks fine otherwise.

Reply:
I will try to make it a source based package and if the build succeeds I will update the PR.